### PR TITLE
Fix Sphinx builds with Sphinx versions != 1.7.5

### PR DESCRIPTION
**Pull request by tpoterba**
Also had to switch PLINK hosting.

### CODE REVIEW

File: hail/python/hail/docs/genetics/index.rst [hunk 1]
```diff
@@ -5,17 +5,15 @@ genetics
...
     :nosignatures:
     :toctree: ./
     :template: class.rst
 
-    Call
-    Locus
-    Pedigree
-    ReferenceGenome
-    Trio
+    hail.Call
```
Comment (tpoterba) [inline hail/python/hail/docs/genetics/index.rst]: This is the bit that fixes the `cannot import GroupedMatrixTable` issue. Obviously.

And it's not even our fault, just seem to be avoiding a sphinx bug

Comment (danking) [reply hail/python/hail/docs/genetics/index.rst]: I'm rather unhappy with removing current module in this one case. Does this mean everything else is silently waiting to break (e.g. linalg)?

File: hail/python/hail/docs/Makefile [hunk 2]
```diff
@@ -47,7 +47,12 @@ help:
 clean:
 	rm -rf $(BUILDDIR)
 
-.PHONY: html
+.PHONY: check-python
+check-python:
```
[the remark this answers is not in the document]
Comment (tpoterba) [reply hail/python/hail/docs/Makefile]: This is intended to be a better error message. If you can't import Hail, you'll still get some other random import error.

**Comment by tpoterba** [comment]
wtf, now it works with a reverted genetics.rst file?

**Comment by tpoterba** [comment]
OK, this is now a PR to fix PLINK.

File: hail/python/hail/docs/Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (danking) [reply hail/python/hail/docs/Makefile]: Do we run this make target anywhere?

[the remark this answers is not in the document]
Comment (tpoterba) [reply hail/python/hail/docs/Makefile]: Oops, I meant to put the `check-python` as a html dependency instead of on the phony line. fixed.

**Pull request merged**